executor/include/execution/executor.hpp

```diff
@@ -7,3 +7,77 @@
 #include <execution/executor/inline_executor.hpp>
 #include <execution/executor/omp_executor.hpp>
 #include <execution/executor/sse_executor.hpp>
+
+TYPE_TO_STRING(inline_executor<oneway_t, single_t, blocking_t::always_t>);
+TYPE_TO_STRING(sse_executor<oneway_t, bulk_t, blocking_t::always_t>);
+TYPE_TO_STRING(omp_executor<oneway_t, bulk_t, blocking_t::always_t>);
+TYPE_TO_STRING(cuda_executor<oneway_t, bulk_t, blocking_t::always_t>);
+
+#define executors                                                \
```

Comment:
`using type = expression;` doesn't work for this?

Reply:
No, since I only need to add CUDA executors only in certain test cases and not all. currently.